contracts/AaveV3Lender.sol

```diff
@@ -363,6 +371,15 @@ contract AaveV3Lender is BaseStrategy, UniswapV3Swapper, AuctionSwapper {
         return (_data & mask) != 0;
     }
 
+    /**
+    @ @dev Gets the liquid balance that can be withdrawn from the pool
+    */
+    function _virtualBalance() internal view returns (uint256) {
+        lendingPool
+            .getReserveDataExtended(address(asset))
+            .virtualUnderlyingBalance;
+    }
```

Comment:
This can be disabled or toggled off:

https://etherscan.io/address/0x34339f94350ec5274ea44d0c37dae9e968c44081#code#F10#L209

https://etherscan.io/address/0x34339f94350ec5274ea44d0c37dae9e968c44081#code#F8#L554

You can perform this check similar to how AAVE logic does it:

https://etherscan.io/address/0x34339f94350ec5274ea44d0c37dae9e968c44081#code#F8#L570

If it's toggled off, use `balanceOf` as the legacy method; otherwise, get the balance from this function.